lib/emoji/Makefile

```
@@ -0,0 +1,44 @@
+BUILD_DIR := build
+
+all: test
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)